### ISSUE TITLE
Not working behind proxy configured

### ISSUE THREAD

**Issue by MagicGJ** [Unthrottled/AMII#91]
I'm getting these two notifications, which seems to be a network problem?
```
notification.no-memes.title="{0}" unavailable offline
notification.no-memes.body=Sorry, I couldn't find a meme. Please try again when online.
```

I have proxy configured, and successfuly downloaded the plugin from the market. So I think IDEA proxy in properly configured.

Is there any extra configurations needed? Or it is just because I cannot access the resource URL with my proxy.

And is it possible to download an offline resource pack or something?

Thanks

**Comment by Unthrottled** [comment]
:thinking: Hmm, I haven't tested anything out with Intellij being configured [to proxy internet connection](https://www.jetbrains.com/help/idea/settings-http-proxy.html). I guess access to https://amii.assets.unthrottled.io/ is being blocked, huh?

> And is it possible to download an offline resource pack or something?

I looked into that, I could upload it to someplace, but it is a 450MB file :sweat_smile: (Didn't realize I had that much content I am giving to users)

**Comment by MagicGJ** [comment]
> 🤔 Hmm, I haven't tested anything out with Intellij being configured [to proxy internet connection](https://www.jetbrains.com/help/idea/settings-http-proxy.html). I guess access to https://amii.assets.unthrottled.io/ is being blocked, huh?
> 
> > And is it possible to download an offline resource pack or something?
> 
> I looked into that, I could upload it to someplace, but it is a 450MB file 😅 (Didn't realize I had that much content I am giving to users)

My browser is configured with the same proxy, and I can access https://amii.assets.unthrottled.io/ properly

**Comment by Unthrottled** [comment]
Interesting, one more thing I want you to try, how about this? (I forgot about the API) https://amii.api.unthrottled.io/public/assets/visuals

I'll eventually get around to trying to replicate the issue.

**Comment by MagicGJ** [comment]
I have access to this one, too. Something like a json string?

That's weired, since I can access these APIs via browser with same proxy...

**Comment by Unthrottled** [comment]
> Something like a json string?
Yes. Well I can rule that you don't have access to those out. I've got some more things that I want to try, but I'll try them out before asking you to try. 

I've got some things I would like to finish first before looking at this. I might have time this weekend to take a peek. Thanks for your patience!

**Comment by MagicGJ** [comment]
Take your time😉

**Comment by MagicGJ** [comment]
Problem solved with #93 😃

**Comment by Unthrottled** [comment]
Awesome! Thank you for following up. I'll have to do this for other projects so they work behind an HTTP proxy.
